docs/deployment_practices.rst

```diff
@@ -278,7 +278,7 @@ the attack surface.
 Minimum requirements for the SecureDrop environment
 ---------------------------------------------------
 
--  The Application and Monitor servers should be dedicated physical
+-  The Application and *Monitor Server*s should be dedicated physical
```

Comment:
You should probably italicize that final 's'. I see this mistake elsewhere too, so do a `rgrep ver\*s` for it.